src/Command/ModifyEntity/LineItem/LineItemChangeDatesCommand.php

```diff
@@ -237,20 +238,22 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
     private function initializeLineItemIdsOption(InputInterface $input): void
     {
-        $lineItemIds = array_filter(
-            explode(',', (string)$input->getOption(self::OPTION_LINE_ITEM_IDS)),
-            static function (string $value): bool {
-                return !empty($value) && (int)$value > 0;
-            }
-        );
+        try {
+            $this->lineItemIds = array_map(
```

Comment:
Remember to change help/docs where it give examples with numeric IDs

Reply:
Also: IMO, a good point to raise a discussion - how _this_ should be released, as 2.smth or _3.smth_? Because for me, at least partially, those changes seem a bit breaking :upside_down_face:

Reply:
This is an interesting question, let's discuss it together with the guys, I have a proposal :)

Reply:
Let's postpone this question until we merge back the integration branch back to develop

Reply:
I think you missed to update the command help (L94-95). There are some numeric ids there

And also the docs:

https://github.com/oat-sa/simple-roster/blob/feature/line-item-uuid/docs/cli/modify-entity-line-item-change-dates-command.md
https://github.com/oat-sa/simple-roster/blob/feature/line-item-uuid/docs/cli/modify-entity-line-item-change-state-command.md

Reply:
Fixed